src/runtime/virtcontainers/mount.go

```diff
@@ -54,33 +54,32 @@ func isSystemMount(m string) bool {
 	return false
 }
 
-func isHostDevice(m string) bool {
+// isHostDevice returns whether the given path is a non-regular file
+// under /dev (or /dev itself) on the host. If os.Stat fails on the
+// file, it returns false plus the error from os.Stat.
+func isHostDevice(m string) (bool, error) {
 	m = filepath.Clean(m)
 	if m == "/dev" {
-		return true
+		return true, nil
 	}
 
 	if strings.HasPrefix(m, "/dev/") {
-		// Check if regular file
 		s, err := os.Stat(m)
-
-		// This should not happen. In case file does not exist let the
-		// error be handled by the agent, simply return false here.
 		if err != nil {
-			return false
+			return false, err
 		}
 
 		if s.Mode().IsRegular() {
-			return false
+			return false, nil
```

Comment:
So here it means that the host regular file in /dev would be shared by virtiofs to guest?